cpp/include/cudf/copying.hpp

```diff
@@ -26,6 +26,12 @@ typedef struct CUstream_st* cudaStream_t;
 
 namespace cudf {
 
+enum mask_allocation_policy {
```

Comment:
The enum itself still needs doxygen block. There's an example here: https://docs.google.com/document/d/10Ff4VZlSsizHdF5r9Zrw7dIwlPD0yWWSmNJ6qter7r8/edit#heading=h.rgm5sgyxoiv6